impeller/aiks/aiks_unittests.cc

```diff
@@ -2056,6 +2056,7 @@ TEST_P(AiksTest, SaveLayerFiltersScaleWithTransform) {
   ASSERT_TRUE(OpenPlaygroundHere(canvas.EndRecordingAsPicture()));
 }
 
+#if IMPELLER_ENABLE_3D
```

Comment:
We should activate this flag in a CI build as part of this patch, otherwise Scene will just bitrot.

Reply:
I've updated the mac unpt build (which AFAIU runs the impeller unittests) to add `--impeller-enable-3d`.